Dockerfile-alpine

```diff
@@ -1,5 +1,5 @@
 FROM postgres:13.0-alpine
```

Comment:
similar to https://github.com/citusdata/docker/pull/212/files#r587341039
let's use 13.2 here

Reply:
Fixed